src/engine/SConscript

```diff
@@ -11,8 +11,9 @@ def scons():
     libraries = ['daos_common_pmem', 'gurt', 'cart', 'vos_srv']
     libraries += ['bio', 'dl', 'uuid', 'pthread', 'abt']
     libraries += ['hwloc', 'pmemobj', 'protobuf-c', 'isal']
+    libraries += ['tbbmalloc_proxy']
 
-    prereqs.require(denv, 'hwloc', 'argobots', 'protobufc', 'isal')
+    prereqs.require(denv, 'hwloc', 'argobots', 'protobufc', 'isal', 'tbbmalloc')
```

Comment:
denv is a local copy of env, but I'd just like to check that this doesn't affect anything other than the engine.

Reply:
It should not affect other. I see it used ony when link daos_engine.